networking/network-optimization.adoc

```diff
@@ -0,0 +1,46 @@
+[id='network-optimization']
+= Network optimization
+include::modules/common-attributes.adoc[]
+:context: networking
+
+toc::[]
+
```

Comment:
When I don't have a heading that starts off an assembly, I like to put a `{nbsp} +` on the line right before the first line of text so that it puts a bit of white space between the TOC's horizontal line and the body text. E.g.,:

```
toc::[]

{nbsp} +
The OpenShift SDN...
```